examples/call-service-panel-example/eslint.config.js

```diff
@@ -0,0 +1,23 @@
+// @ts-check
+
+const foxglove = require("@foxglove/eslint-plugin");
+const globals = require("globals");
+const tseslint = require("typescript-eslint");
+
+module.exports = tseslint.config({
+  files: ["src/**/*.ts", "src/**/*.tsx"],
```

Comment:
I think putting this `files` restriction at the top level may mean that this `eslint.config.js` file itself is not linted (and any other `.js` files). Would be interesting to confirm that. Maybe not the end of the world but it feels nice to lint all JS & TS files.